README.md

```diff
@@ -137,18 +142,22 @@ sh 4.run_sbs_phenotype_processes.sh
 ```
 **Slurm**:
 ```sh
-sh 4.run_sbs_phenotype_processes_slurm.sh
+sbatch 4.run_sbs_phenotype_processes_slurm.sh
 ```
 
 ***Note**: Use `brieflow_configuration` Conda environment for each configuration notebook.
 
+***Note**: Many users will want to only run SBS or phenotype processing, independently.
+By varying the tags in the .sh files (`--until all_sbs_process` or `--until all_phenotype_process`), the analysis will only run only the analysis of interest.
```

Comment:
I would align this explanation to the prior:
1) if there are no files, it won't run
2) if you want to manually restrict it from running, use the until flag

Reply:
Done in https://github.com/cheeseman-lab/brieflow/pull/29/commits/4cf230dd1bb9f2324e714f0950c08a582f1ed673